src/LinAlg/hiopVectorRajaImpl.hpp

```diff
@@ -2024,14 +2073,11 @@ bool hiopVectorRajaPar::isfinite_local() const
  * 
  * @pre Vector data was moved from the memory space to the host mirror.
  */
-void hiopVectorRajaPar::print(FILE* file/*=nullptr*/, const char* msg/*=nullptr*/, int max_elems/*=-1*/, int rank/*=-1*/) const
+template<class MEM, class POL>
+void hiopVectorRaja<MEM,POL>::
+print(FILE* file, const char* msg/*=NULL*/, int max_elems/*=-1*/, int rank/*=-1*/) const
```

Comment:
These default values should be enforced either in header file or here... I commented on another function that had default values removed, so wanted to make sure that functionality wasn't lost.